src/Common/tests/TestUtilities/XUnit/UseCultureAttribute.cs

```diff
@@ -21,8 +21,8 @@ public class UseCultureAttribute : BeforeAfterTestAttribute
     {
         private readonly Lazy<CultureInfo> _culture;
         private readonly Lazy<CultureInfo> _uiCulture;
-        private CultureInfo _originalCulture;
-        private CultureInfo _originalUICulture;
+        private CultureInfo? _originalCulture;
+        private CultureInfo? _originalUICulture;
```

Comment:
I don't believe either of these can ever be null:
```cs
            _originalCulture = Thread.CurrentThread.CurrentCulture;
            _originalUICulture = Thread.CurrentThread.CurrentUICulture;
```
```suggestion
        private CultureInfo _originalCulture = null!;
        private CultureInfo _originalUICulture = null!;
```

Reply:
https://learn.microsoft.com/en-us/dotnet/api/system.threading.thread.currentuiculture?view=net-7.0#exceptions

Reply:
Thank you! I wasn't quite sure the best way to get around the fact that these are not set in the constructor. It makes more sense that these do not have `?`